setup.py

```diff
@@ -179,9 +179,9 @@ def package_files(prefix: str, directory: str, extension: str):
 ]
 
 extra_deps['nlp'] = [
-    'transformers>=4.11,!=4.34.0,<4.42',
+    'transformers>=4.11,!=4.34.0,<4.43',
```

Comment:
i'm assuming there's some dark reason we don't use 4.34.0...

Reply:
It was very broken lol https://github.com/mosaicml/composer/pull/2635